ibis/backends/dask/tests/test_udf.py

```diff
@@ -0,0 +1,257 @@
+import dask.dataframe as dd
+import numpy as np
+import pandas as pd
+import pandas.testing as tm
+import pytest
+
+import ibis
+import ibis.expr.datatypes as dt
+import ibis.expr.types as ir
+
+from .. import connect
+from ..udf import nullable, udf
+
+
+# TODO - move these to the top level conftest?
+# Why does pandas keep them here
+@pytest.fixture
+def df(npartitions):
```

Comment:
There is shared test under https://github.com/ibis-project/ibis/blob/master/ibis/backends/tests/test_vectorized_udf.py